tests/zoomus/test_client.py

```diff
@@ -47,7 +47,17 @@ def test_cannot_init_with_non_int_timeout(self):
     def test_init_creates_all_components(self):
         client = ZoomClient("KEY", "SECRET")
         self.assertEqual(
-            set(["meeting", "past_meeting", "report", "user", "webinar", "recording"]),
+            set(
+                [
+                    "meeting",
+                    "past_meeting",
+                    "report",
+                    "user",
+                    "webinar",
+                    "recording",
+                    "live_stream",
```

Comment:
There looks to be a trailing ',' at the end of this set. This would fail a run of `black`, so please make sure to run `black` to make sure that all the formatting things are taken care of. Thanks!

Reply:
This is fixed